keep/api/models/db/migrations/versions/2024-08-09-10-53_6e353161f5a8.py

```diff
@@ -0,0 +1,21 @@
+"""Merge
+
+Revision ID: 6e353161f5a8
+Revises: c91b348b94f2, 42098785763c
+Create Date: 2024-08-09 10:53:33.363763
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "6e353161f5a8"
+down_revision = ("c91b348b94f2", "42098785763c")
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    pass
```

Comment:
?

Reply:
It's a merge migration to join multiple heads, best practice for alembic to ensure all environments work (including those who migrated to one of the branches): https://stackoverflow.com/questions/22342643/alembic-revision-multiple-heads-due-branching-error